app/models/journal.rb

```diff
@@ -1,2 +1,14 @@
 class Journal < ApplicationRecord
+  belongs_to :user
+
+  validates :title, presence: true
+  validates :date, presence: true
+  validates :content, presence: true
+  validates :goals_tomorrow, presence: true
+  validates :goals_today, presence: true
```

Comment:
Let's add null: false db level validations through a migration for all these model validations